locales/en.js

```diff
@@ -0,0 +1,55 @@
+// Example translation dictionary. The keys stay the same in each translation, and the values are the desired translation.
+
+const translations = {
+  // errors
+  // commander.missingArgument
+  "error: missing required argument '{0}'": "error: missing required argument '{0}'",
+  // commander.optionMissingArgument
+  "error: option '{0}' argument missing": "error: option '{0}' argument missing",
+  // commander.missingMandatoryOptionValue
+  "error: required option '{0}' not specified": "error: required option '{0}' not specified",
+  // commander.conflictingOption
+  'error: {0} cannot be used with {1}': 'error: {0} cannot be used with {1}',
```

Comment:
Can change the template strings and substitutions for it, or provide a way to modify it by translater?
The word `option` seems to be hard-code in substitution, which will course some translations problem:
```js
// en
// $ program answer --yes --no
// error: option '--no' cannot be used with option '--yes'
// zh-CN
// $ program answer --yes --no
// 错误: 选项 option '--no' 不能与 option '--yes' 共同使用
//            ^                   ^

Reply:
I missed that one. Message being constructed with two values that may come from CLI option arguments or may come from environment variable. In `getErrorMessage`.

https://github.com/tj/commander.js/blob/0b4198dde02cc8fc571c1265afa5bde447f6fca8/lib/command.js#L1682

```
      if (source === 'env') {
        return `environment variable '${bestOption.envVar}'`;
      }
      return `option '${bestOption.flags}'`;
```

Reply:
I have reworked the error message to remove the hard-coded `option` and `environment variable` strings in the code. The environment variable is now listed after the option, using the same translation as in the description of the option in the help.

I did a simple update to `zh-CN` but it may need further changes. It displays error messages like:

```
$ program answer --right --wrong
错误: 选项 '--wrong' 不能与 '--right' 共同使用

-----

$ RIGHT=1 program answer --wrong
错误: 选项 '--wrong' 不能与 '--right' (所需环境变量: RIGHT) 共同使用
```

Reply:
It looks great now.

Reply:
👍